lib/repl.js

```diff
@@ -928,6 +951,16 @@ function complete(line, callback) {
   function completionGroupsLoaded(err) {
     if (err) throw err;
 
+    if (warning.type) {
+      self._writeToOutput('\r\n');
+      process.emitWarning(
+        warning.message,
+        warning.type,
+        undefined,
+        undefined,
+        true);
```

Comment:
@jasnell, For aesthetics.

With last parameter `false`:

![image](https://user-images.githubusercontent.com/2842176/27342096-b4eb227e-5611-11e7-9ef4-b286853a840c.png)

With immediately:

![image](https://user-images.githubusercontent.com/2842176/27342127-cd3f5e94-5611-11e7-947a-6f05aac10160.png)